### PR TITLE
feat: auto detect css module based filename

### PR DESCRIPTION
**Pull request by JiangWeixian**
similar options like https://github.com/webpack-contrib/css-loader#auto

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,6 +31,9 @@ postcssLightningcss({
   // Use a browserslist query that will inform which browsers are supported
   // Will add or remove vendor prefixes that are needed or not anymore
   browsers: ">= .25%",
+  // Auto set cssModules to true if the file matches this regex
+  // Will be overriden by the lightningcssOptions cssModules(boolean) option
+  cssModulesRe: /\.module\.css/,
```
Comment (onigoetz) [inline README.md:36]: I have the feeling this option should be renamed to `cssModules` and support the following options: 
- boolean:  as it supports today
- `"auto"`: uses the `/\.module\.\w+$/i` regex to decide
- `RegEx`: uses a custom RegEx

With the default being `false` to introduce no breaking change

Can you please make this change ?

**Review by onigoetz — CHANGES_REQUESTED** [review]
Hi, thanks for the PR!

I've added a comment on the name of the option; what do you think of my proposition?

File: README.md (hunk 1 above)
Comment (JiangWeixian) [reply README.md:36]: Sure!

**Comment by onigoetz** [comment]
This is released in 0.7.0 : https://github.com/onigoetz/postcss-lightningcss/releases/tag/0.7.0

thanks again for the contribution :)

**Pull request merged**